src/raftstore/store/region_snapshot.rs

```diff
@@ -38,13 +42,39 @@ impl RegionSnapshot {
         RegionSnapshot {
             snap,
             region: Arc::new(region),
+            apply_index: Arc::new(AtomicU64::new(std::u64::MAX)),
         }
     }
 
+    #[inline]
     pub fn get_region(&self) -> &Region {
         &self.region
     }
 
+    #[inline]
+    pub fn get_apply_index(&self) -> Result<u64> {
+        let apply_index = self.apply_index.load(Ordering::SeqCst);
+        if apply_index == std::u64::MAX {
```

Comment:
I prefer to use 0 as default value, it's ok because the min value is 5.

Reply:
Thanks for the suggestion!